canonical-web-and-design/canonical.com
# Add location filter to careers pages

## Done

Added the ability to filter jobs by location on the careers pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/all
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that jobs are properly filtered by location, and that the new field is compatible with the existing "Filter by" and "Sort by" fields.
- Ensure that the code is sensible and in line with any Canonical style guides

## Issue / Card

Fixes #189
